### ISSUE TITLE
Merge all map signals into one

### ISSUE THREAD

**Issue by nyurik** [nyurik/kibana-vega-vis#14]
I suspect it will be more efficient for the two-way binding to have a single object `{lat, lon, zoom}` rather than three individual signals.  Also need an example of a graph that moves the map on an event.

**Comment by emilmirzayev** [comment]
Hi Yuri,

when the new plugin release for ELK 6.2x with map support will be received? Can you provide with any dates or so?

thanks

**Comment by nyurik** [comment]
@emilmirzayev good question - I hope I can work on it very soon - I need to get the 6.3 changes out of the door first.  Are you trying to use interactive map base layer, or are you using the standard Vega map support?

**Comment by emilmirzayev** [comment]
Hi Yuri,
thanks for your answer. First, I want to say that, I am using Vega for 2 weeks now, and I like it very much. Vega-lite is also a good option, but I prefer control.
My goal is, to combine the map layer in which i have my devices drawn, and the WMS layer I made where I map transformers. I tried it with Kibanas own map + WMS, but it occurred to be somewhat ugly. I will also paste them below as images. This is how it looks without WMS
![image](https://user-images.githubusercontent.com/25964049/38351786-52613d86-38b1-11e8-87dd-a5b5b44c2bda.png)
This is how it looks with WMS
![image](https://user-images.githubusercontent.com/25964049/38351800-62c0a022-38b1-11e8-8c78-5095cc16d19f.png)
Maybe there are other ways or ideas which I can use? I am not very experienced in geo visualizations, that's why, every help/hint is appreciated
